packit/distgit.py

```diff
@@ -186,7 +186,8 @@ def clone_package(
         Clone package from dist-git, i.e. from:
         - Fedora's src.[stg.]fedoraproject.org
         - CentOS Stream's gitlab.com/redhat/centos-stream/rpms/
-        depending on configured pkg_tool: {fedpkg(default),fedpkg-stage,centpkg}
+        - CentOS CBS's gitlab.com/CentOS/<SIG>/rpms/
```

Comment:
on Gitlab, repos are namespaced by SIG.

Alternatively, CBS can consume dist-git repos from the https://git.centos.org/rpms/ Pagure instance namespace